custom_components/hyxi_cloud/__init__.py

```
@@ -375,7 +375,134 @@ async def _async_resolve_webhook_url(
+            if coordinator.subscribe_code:
+                await async_register_subscription_code(hass, coordinator.subscribe_code)
+            _LOGGER.info(
+                "Successfully subscribed to HYXI Real-Time Push (code: %s)",
+                coordinator.subscribe_code,
+            )
+        else:
+            coordinator.push_status = "error"
+            msg = res.get("msg", "Unknown error")
+            coordinator.push_error = msg
+            _log_push_subscription_failure("HYXI Real-Time Push", msg)
+    except Exception as err:  # pylint: disable=broad-exception-caught
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)